metric/config.go

```diff
@@ -3,7 +3,9 @@
 
 package metric // import "go.opentelemetry.io/otel/metric"
 
-import "go.opentelemetry.io/otel/attribute"
+import (
+	"go.opentelemetry.io/otel/attribute"
+)
```

Comment:
Why? Can we revert this?

Reply:
Ah sorry, I think I had imported slices at some point and then removed it, and goimports changed - will revert.